main.go

```diff
@@ -117,13 +117,11 @@ func cmdServer(args *cli.Context) error {
 			3)
 	}
 
-	if args.Bool("automigrate") {
-		err = mongo.Migrate(context.Background(), mongo.DbVersion, dbSession, true)
-		if err != nil {
-			return cli.NewExitError(
-				fmt.Sprintf("failed to run migrations: %v", err),
-				3)
-		}
+	err = mongo.Migrate(context.Background(), mongo.DbVersion, dbSession, true)
```

Comment:
this way we'll always run migrations (not only check db version if automigrate is off)
we still need to handle automigrate command
it should look like
if automigrate 
mongo.Migrate(context.Background(), mongo.DbVersion, dbSession, true)
else
mongo.Migrate(context.Background(), mongo.DbVersion, dbSession, false)

Reply:
> this way we'll always run migrations (not only check db version if automigrate is off)
> we still need to handle automigrate command
> it should look like
> if automigrate
> mongo.Migrate(context.Background(), mongo.DbVersion, dbSession, true)
> else
> mongo.Migrate(context.Background(), mongo.DbVersion, dbSession, false)

ah yes! for got the last argument. [fixed](https://github.com/mendersoftware/deployments/pull/359/commits/326c66a1679ac194d0e22ef098b38e07ccc890aa)

Reply:
the deployments have different db.Migrate thats why I missed that.